src/blueprints/tv2/action-factories/tv2-graphics-action-factory.ts

```diff
@@ -422,20 +422,24 @@ export class Tv2GraphicsActionFactory {
 
     const enable: TimelineEnable = { start: blueprintConfiguration.studio.CasparPrerollDuration }
     const sourceInput: number = this.getDownstreamKeyerMatchingRole(blueprintConfiguration, Tv2DownstreamKeyerRole.FULL_GRAPHICS).Fill
-    const transition: AtemTransition = AtemTransition.WIPE
-    const transitionSettings: AtemTransitionSettings = {
+    const transitionType: VideoMixerTransition = VideoMixerTransition.WIPE
+    const transitionSettings: VideoMixerTransitionSettings = {
       wipe: {
-        rate: blueprintConfiguration.studio.HTMLGraphics.TransitionSettings.wipeRate,
+        frameRate: blueprintConfiguration.studio.HTMLGraphics.TransitionSettings.wipeRate,
         pattern: 1,
         reverseDirection: true,
```

Comment:
Just a note: Instead of specifying whether to reverse the wipe direction, then it would be easier as a developer and user (if this is at some point exposed as a configuration option), to specify the wipe direction instead of trying to guess what the default direction is for an arbitrary video mixer :)